functions/Restore-DbaDatabase.ps1

```diff
@@ -509,7 +509,7 @@ function Restore-DbaDatabase {
 		
         if ($null -ne $DatabaseName) {
             If (($DatabaseName -in ($server.Databases.name)) -and ($WithReplace -eq $false)) {
-                Write-Warning "$FunctionName - $DatabaseName exists on Sql Instance $SqlInstance , must specify WithReplace to continue"
+                Write-Message -Level Warning -Message "$FunctionName - $DatabaseName exists on Sql Instance $SqlInstance , must specify WithReplace to continue" 
                 break
```

Comment:
Use Stop-Function instead. Including the function name in messages is no longer needed.